homeassistant/components/apple_tv/strings.json

```
@@ -0,0 +1,68 @@
+{
+  "config": {
+    "title": "Apple TV",
+    "flow_title": "Apple TV: {name}",
+    "step": {
+      "user": {
+        "title": "Setup a new Apple TV",
+        "description": "Start by entering the device name (e.g. Kitchen or Bedroom) or IP-address of the Apple TV you want to add. If any devices were automatically found on your network, they are shown below.\n\nIf you cannot see your device or experience any issues, try specifying the device IP-address.\n\n{devices}",
```

Fixed a typo of [IP address](https://www.dictionary.com/browse/ip-address):
```suggestion
        "description": "Start by entering the device name (e.g. Kitchen or Bedroom) or IP address of the Apple TV you want to add. If any devices were automatically found on your network, they are shown below.\n\nIf you cannot see your device or experience any issues, try specifying the device IP address.\n\n{devices}",
```